workflow/info.plist

```diff
@@ -4,11 +4,9 @@
 <dict>
 	<key>bundleid</key>
 	<string>ws.willner.alfred.imageoptim</string>
-	<key>category</key>
-	<string>Tools</string>
```

Comment:
why removing these lines?

Reply:
Nice catch, it wasn’t on purpose. I don’t use categories, so it must’ve been auto-removed. Pushing a fix now.